test/end2end_test.go

```diff
@@ -4991,6 +4991,10 @@ type stubServer struct {
 	cc     *grpc.ClientConn
 	s      *grpc.Server
 
+	network string
+	address string
```

Comment:
Is this the same as `addr` (immediately below)?

Reply:
`addr` is the address pulled from the listener after `net.Listen` to build the target if necessary (IE not in the unix cases), and `address` is the parameter used in `net.Listen`.

Reply:
So they're mostly the same thing and they have extremely similar names.  This is a recipe for confusion.  Do we need both?

Reply:
No, got rid of addr and replaced with address.